src/modules/element.js

```diff
@@ -1,3 +1,9 @@
-const scoreArray = [100, 50, 30, 40, 20, 10];
+import url from './url.js';
 
-export default scoreArray;
\ No newline at end of file
+const displayBody = document.querySelector('.display_body');
+
+export default async function fetchDATA() {
+  const response = await fetch(url);
+  const data = await response.json();
+  displayBody.innerHTML = data.result.map((res) => `<li>${res.user}: ${res.score}</li>`).join('');
+}
```

Comment:
- [ ] **Optional:**
- You could make your leaderboard render the list on the first load. This is a better experience for users to see data displayed from your API.